layout-final/individual.html

```diff
@@ -1,112 +1,111 @@
-				</p>
-				<a href="">Leia Mais</a>
-			</div> <!-- post closure -->
-			
-			
-					<!-- post opening -->
-			<div class="post">
-				<h2>CRUNCH TACOS </h2>
-				<img width="620px" src="image/image4.png">
-				<p>
-				3 deliciosos tacos montados na tortilla crocante de milho, com queijo, alface, pico de gallo e servidos com a nossa guacamole.
-				</p>
-				<a href="">Leia Mais</a>
-			</div> <!-- post closure -->
-
-						<!-- post opening -->
-			<div class="post">
-				<h2>TIJUANA TACOS </h2>
-				<img  width="620px" src="image/image5.jpg">
-					<p>
-					3 tacos recheados com carne grelhada, bacon, queijo, onion crisps e pico de gallo. Acompanha nossa maionese.
-					</p>
-				<a href="">Leia Mais</a>
-			</div> <!-- post closure -->
-
-						<!-- post opening -->
-			<div class="post">
-				<h2>QUESADILLAS </h2>
-				
-				<img width="620px" src="image/image.png">
-				<p>
-					Nossas incomparáveis tortillas caseiras tostadas e recheadas com queijo derretido e pico de gallo. Acompanha guacamole.
-				</p>
-				<a href="">Leia Mais</a>
-			</div> <!-- post closure -->
-
-		</div>
-
-	
-
-			
-		<div class="side-content">
-			<h3>Individuais</h3>
-
-			<a href="quesadillas.html">Quesadillas</a>
-			<a href="burritos.html">Burritos</a>
-			<a href="crunch-tacos.html">Crunch Tacos</a>
-			<a href="tijuana-tacos.html">Tijuana Tacos</a>				
-		</div>
-	</div>
-
-
-	<div id="footer">
-		Todos os direitos reservados
-		
-	</div>
-
-	</div>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <title>La Creditas Restaurant</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="header">
+      <div id="area-logo">
+        <h1>
+          <span class="green">La</span><span class="white">Creditas</span
+          ><span class="red">Restaurant</span>
+        </h1>
+      </div>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="individual.html">individual</a></li>
+          <li><a href="combo-premium.html">Combo Premium</a></li>
+          <li><a href="who-we-are.html">Quem Somos</a></li>
+          <li><a href="contact.html">Contato</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <div class="post">
+          <h2>BURRITOS</h2>
+
+          <picture>
+            <source srcset="image/image3.png" />
+            <img src="/image/imag3.png" alt="prato de comida mexicana" />
+          </picture>
+
+          <p>
+            Burrito recheado com alface, pico de gallo, sour cream, queijo e
+            guacamole. Acompanha salsa da casa.
+          </p>
+          <a href="">Leia Mais</a>

```

Comment:
Colocar uma cerquilha como valor do href em links que ainda não tem endereços pode evitar a atualização da página ao clicar nele.
```suggestion
          <a href="#">Leia Mais</a>

```

Reply:
```suggestion
          <a href="">Leia Mais</a>

```